app/src/main/res/menu/login_delete.xml

```diff
@@ -1,14 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we delete this whole file? I don't think we're using it anymore?

Reply:
No, with the feature flag we want to keep this